Cargo.toml

```diff
@@ -81,5 +85,5 @@ crate-type = ["bin"]
 
 [[test]]
 name = "trust_metric"
-path = "tests/trust_metric.rs"
+path = "tests/tests.rs"
```

Comment:
Why did you change the path here?

Reply:
The name `trust_metric` is not suitable, and I rename it.

Reply:
@zeroqn

Reply:
IMO, ```test.rs``` means a general test, but ```[[test]]``` here means a specific
integration test.

Each integration test results in a separate executable binary. 
reference: https://doc.rust-lang.org/cargo/reference/cargo-targets.html#tests